hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieMergeOnReadRDD.scala

```diff
@@ -118,6 +123,18 @@ class HoodieMergeOnReadRDD(@transient sc: SparkContext,
     rows
   }
 
+  private def extractRequiredSchema(iter: Iterator[InternalRow]): Iterator[InternalRow] = {
```

Comment:
the name could imply returning schema

```suggestion
  private def toRowsWithRequiredSchema(iter: Iterator[InternalRow]): Iterator[InternalRow] = {
```